chart/values.yaml

```diff
@@ -0,0 +1,13 @@
+ingress-svc:
+  name: traefik-ingress-service
+  namespace: kube-system
+
+image:
+  repository: pennlabs/ingress-bouncer
+  pullPolicy: IfNotPresent
+  tag: latest
```

Comment:
Can we somehow specify to use the sha of the current commit?

Reply:
nah that shouldn't be the job of the chart anyway. the chart should specify latest, and the user should specify if they want to override

Reply:
Okay, makes sense